README.md

```diff
@@ -48,20 +60,24 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
     <input type="submit" value="Submit" id="submit">
   </form>
 ```
+var user_input = $('#submit').val();
```

Comment:
You're on the right track, but the submit button doesn't contain what the user typed. The text box with ID of color does. So that would be `var user_input = $('#color').val()`.